### ISSUE TITLE
Retain color within child process

### ISSUE THREAD

**Issue by Aaronius** [1337programming/webpack-shell-plugin#17]
First of all, thanks for the plugin. It's very helpful.

Currently, child processes can't output colored text which makes me :(. I did a little googling and it looks like most people are recommending [using `spawn`](http://stackoverflow.com/a/14231570/459966) instead of `exec`. I haven't dug into it beyond that.

**Comment by mikeerickson** [comment]
@Aaronius Have you found a workaround for this issue?

**Comment by Aaronius** [comment]
Negative. Sad face.

**Comment by mikeerickson** [comment]
@Aaronius Thx for reply :-)   I saw there was a pull request for this, however, it doesnt appear to be merged -- https://github.com/1337programming/webpack-shell-plugin/pull/19

**Comment by opiepj** [comment]
I looked into this issue, I choose to use exec instead of spawn because exec is universally consistent across operating systems. I've found spawn tends to lead to a lot of issues as you have to worry more on OS architecture requirements. Thoughts?

**Comment by evisong** [comment]
What about doing a OS check and only switch to spawn on certain OS's? The preference of color output is because developers're watching the console when coding, so I think it's OK to tune an enhanced experience to only certain OS which developers work on. 

According to http://stackoverflow.com/research/developer-survey-2016#technology-desktop-operating-system , macOS (Mac OS X) has a **26.2%** market share among developers, so I personally wish to support spawn only for macOS.

Same as others, I'd like to say thank you for this handy plugin. 😄

**Comment by mikeerickson** [comment]
So, I was in the process of creating the repo to show my issue, and in that process, I found the issue :-)

I was still requiring the original webapck-shell-plugin

```
const WebpackShellPlugin = require('webpack-shell-plugin');
```
now
```
const WebpackShellPlugin = require('@slightlytyler/webpack-shell-plugin');
```

And all is good now (shell output in color as expected)

**Comment by opiepj** [comment]
Merged #19

**Comment by mariotacke** [comment]
Was this ever figured out? The PR (#19) seems to have started a discussion but it kind of left open ended. I am on Windows / plugin version 0.5.0. @slightlytyler @opiepj
